examples/connectivity/plot_connectivity_measures.py

```diff
@@ -0,0 +1,92 @@
+"""
+Comparing different connectivity measures
+=========================================
+
+This example compares different measures of functional connectivity between
+regions of interest.
+
+"""
+
+# Fetch dataset
+import nilearn.datasets
+atlas = nilearn.datasets.fetch_atlas_msdl()
+dataset = nilearn.datasets.fetch_adhd()
+
+# Extract regions time series
+import nilearn.input_data
+masker = nilearn.input_data.NiftiMapsMasker(
+    atlas.maps, resampling_target="maps", detrend=True,
+    low_pass=None, high_pass=None, t_r=2.5, standardize=False,
+    memory='nilearn_cache', memory_level=1)
+subjects = []
+for func_file in dataset.func:
+    time_series = masker.fit_transform(func_file)
+    subjects.append(time_series)
+
+# Estimate connectivity
+import nilearn.connectivity
+kinds = ['robust dispersion', 'partial correlation', 'precision',
```

Comment:
It would be nice if we can strict to showing only measures to partial correlation, covariances, correlation. This way we can also save some time in running.